app/services/data_migrations/backfill_equality_and_diversity_completed_attributes.rb

```diff
@@ -0,0 +1,20 @@
+module DataMigrations
+  class BackfillEqualityAndDiversityCompletedAttributes
+    TIMESTAMP = 20230606115559
+    MANUAL_RUN = true
+
+    def change
+      applications.find_each(batch_size: 500) do |application|
+        application.update_columns(equality_and_diversity_completed: true, equality_and_diversity_completed_at: application.submitted_at)
+      end
+    end
+
+  private
+
+    def applications
+      ApplicationForm
+        .where.not(equality_and_diversity: nil)
+        .where.not(submitted_at: nil)
```

Comment:
I'd recommend adding a where not `equality_and_diversity_completed: true` In case the manual run "fails" at the middle and you need to run again. Then you won't touch the ones that you updated on the last run. Just a safety mechanism.

Reply:
👌

Reply:
the behaviour of this was slightly funny:
```
[16] pry(#<DataMigrations::BackfillEqualityAndDiversityCompletedAttributes>)> ApplicationForm.where(equality_and_diversity_completed: nil).count
=> 1
[17] pry(#<DataMigrations::BackfillEqualityAndDiversityCompletedAttributes>)> ApplicationForm.where.not(equality_and_diversity_completed: true).count
=> 0
```

So I've added `.where(equality_and_diversity_completed: nil)` instead